data/settings/src/main/proto/com/google/jetpackcamera/settings/jca_settings.proto

```diff
@@ -31,19 +31,19 @@ option java_multiple_files = true;
 
 message JcaSettings {
   bool default_front_camera = 2 [deprecated = true];
-  bool front_camera_available = 3;
-  bool back_camera_available = 4;
+  bool front_camera_available = 3 [deprecated = true];
```

Comment:
Do we want to keep these as deprecated? Since we're pre-release, would it be better to get rid of these values?

Reply:
Yeah I'll remove them. 

After release, we will need to keep track and make sure that we don't remove/renumber whenever we update the build code of the app.